File: week3-exercises02-scala/src/test/scala/com/datastax/workshop/Ex11_ClearTables.scala
package com.datastax.workshop

import org.junit.jupiter.api.Test
import org.junit.platform.runner.JUnitPlatform
import org.junit.runner.RunWith

@RunWith(classOf[JUnitPlatform])
object Ex11_ClearTables extends ExerciseBase("Exercise4")

@RunWith(classOf[JUnitPlatform])
class Ex11_ClearTables {

  import Ex11_ClearTables._

  @Test
  def clear_tables(): Unit = {

    journeyRepo.clearTables()

    LOGGER.info("SUCCESS")
  }
}

File: week3-exercises01-java/src/test/java/com/datastax/workshop/Ex11_ClearTables.java
package com.datastax.workshop;

import org.junit.jupiter.api.Test;
import org.junit.platform.runner.JUnitPlatform;
import org.junit.runner.RunWith;

@RunWith(JUnitPlatform.class)
public class Ex11_ClearTables extends ExerciseBase {

    public Ex11_ClearTables() {
        super("Exercise4");
    }

    @Test
    public void clear_tables() {

        journeyRepo.clearTables();

        LOGGER.info("SUCCESS");
    }
}
